config/params.php

```diff
@@ -44,11 +54,29 @@
         ],
     ],
 
+    'yiisoft/mailer' => [
```

Comment:
That should be separated into two configs:

1. Generic mailer config.
2. Contact-form specific config.